lib/src/types/barcode.dart

```diff
@@ -8,9 +8,9 @@ import 'barcode_format.dart';
 class Barcode {
   Barcode(this.code, this.format, this.rawBytes);
 
-  final String code;
+  final String? code;
```

Comment:
Can `Barcode` be instantiated with `code` having a `null` value?
I found 1 instantiation in `qr_code_scanner.dart` and `code` there is always non-null.

Reply:
Changed to non nullable